components/org.wso2.carbon.identity.data.publisher.audit.idp.properties/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.wso2.carbon.identity.data.publisher.audit</groupId>
         <artifactId>identity-data-publisher-audit</artifactId>
-        <version>1.0.4-SNAPSHOT</version>
+        <version>1.0.3</version>
```

Comment:
Shouldn't we keep this snapshot ?.  Only the release build will change this and update to the snapshot of the next version.